velox/type/Timestamp.h

```diff
@@ -101,7 +101,7 @@ struct Timestamp {
 
   constexpr Timestamp() : seconds_(0), nanos_(0) {}
 
-  Timestamp(int64_t seconds, uint64_t nanos)
+  Timestamp(int64_t seconds, int64_t nanos)
       : seconds_(seconds), nanos_(nanos) {
```

Comment:
`nanos_` is still uint64_t, do we need to change it?

Reply:
Velox's nanos_ is always >= 0.

Reply:
```
seconds += (nanos / kNanosPerSecond - 1);
nanos -= (nanos / kNanosPerSecond - 1) * kNanosPerSecond;
```
nanos will be >= 0 after `-= (nanos / kNanosPerSecond - 1) * kNanosPerSecond`.